tutorials/text/annotations.py

```diff
@@ -98,14 +120,34 @@
 # Do not proceed unless you have already read :ref:`annotations-tutorial`,
 # :func:`~matplotlib.pyplot.text` and :func:`~matplotlib.pyplot.annotate`!
```

Comment:
This line probably needs to go below the new subsection, since I _think_ it's talking about the advanced section.